index-import.cpp

```diff
@@ -99,16 +112,100 @@ static const FileEntry *getFileEntry(FileManager &fileMgr, StringRef path) {
     return fileMgr.getVirtualFile(path, /*size*/ 0, /*modtime*/ 0);
   } else {
     // getVirtualFile asserts and fails when called with the empty string.
-    return fileMgr.getFile(path);
+    return new FileEntry();
```

Comment:
what is this change for?

Reply:
On my machine it failed to compile without it.

Reply:
I am building a new release for Swift 5.2.4, here's the fix I did for this issue: https://github.com/lyft/index-import/pull/48

Reply:
I'll rebase soon to remove my commit.

Reply:
merged

Reply:
rebased